examples/client/Locomotion/src/Components/InformationCard/Card.tsx

```diff
@@ -40,7 +40,7 @@ const Card = ({
           </>
         )}
       </CardTitleContainer>
-      {children ? <CardText>{children}</CardText> : undefined}
+      {children ? <CardText numberOfLines={1}>{children}</CardText> : undefined}
```

Comment:
why? i use it in price breakdown and not sure i want it

Reply:
also used in account, contact us, and credit card details
did we test them all?